_bookdown.yml

```diff
@@ -2,21 +2,32 @@ new_session: yes
 
 rmd_files: [
   "index.Rmd",
+  
   "01-software-modeling.Rmd",
   "02-tidyverse.Rmd",
   "03-base-r.Rmd",
+  
   "04-ames.Rmd",
   "05-data-spending.Rmd",
   "06-feature-engineering.Rmd",
   "07-fitting-models.Rmd",
   "08-the-model-workflow.Rmd",
   "09-judging-model-effectiveness.Rmd",
+  
   "10-resampling.Rmd",
   "11-comparing-models.Rmd",
   "12-tuning-parameters.Rmd",
   "13-grid-search.Rmd",
   "14-iterative-search.Rmd",
   "15-workflow-sets.Rmd",
+  
+  "16-encoding-categorical-data.Rmd",
+  "17-dimensionality-reduction.Rmd",
+  "18-explaining-models-and-predictions.Rmd",
+  "19-when-should-you-trust-predictions.Rmd",
+  "20-ensemble-models.Rmd",
+  "21-inferential-analysis.Rmd",
```

Comment:
I like this order a lot overall; I'm a little uncertain how the flow will go from current Ch 15 back into preprocessing details like big time encoding details, but I think we should go with it for now and reassess as we have more details.